python/lsst/sconsUtils/dependencies.py

```diff
@@ -504,7 +504,6 @@ def _tryImport(self, name):
                 else:
                     module.config.addCustomTests(self.customTests)
                 return module
-        state.log.info("Failed to import configuration for optional package '%s'." % name)
```

Comment:
I'm just guessing here, but I think it's here and it refers to "optional" packages just because there's an *additional* fatal-level log message (in `PackageTree.__init__`) that will appear after this one if there's a failure to load a required package, but this is the only one you see if it's a failure for an optional dependency.